app/models/occupation_standard.rb

```diff
@@ -120,6 +120,12 @@ def ojt_type_display
     ojt_type&.titleize
   end
 
+  def show_national_occupational_framework_badge?
+    national_occupational_framework? &&
+      organization_id.present? &&
+      organization_id == Organization&.urban_institute&.id
```

Comment:
```suggestion
      organization_id == Organization.urban_institute&.id
```
I don't think the first `&` is necessary?